client/app/hosting/module/MODULE.html

```diff
@@ -1,9 +1,5 @@
 <div class="container-fluid px-0" data-ng-controller="HostingTabModulesController as ctrlHostingTabModules">
 
-    <div class="text-center" data-ng-if="ctrlHostingTabModules.loading">
-        <oui-spinner data-size="l"></oui-spinner>
-    </div>
-
     <div class="row" data-ng-show="!ctrlHostingTabModules.loading">
```

Comment:
If the spinner has been removed, I think there is no need for the `ng-show` attribute :)